dlk/python/dlk/core/operators.py

```diff
@@ -2349,7 +2349,7 @@ def infer_shape(cls, lists: Dict[str, List[int]], format: str, input_formats: Li
         return output_shape
 
 
-class QTZ_binary_channel_wise_mean_scaling(Quantizer):
+class QuantizeBinaryChannelWiseMeanScaling(Quantizer):
```

Comment:
As a operator name, it might be room to improve. It is a little uncomfortable that this operator's name starts with a verb. How about `BinaryChannelWiseMeanScalingQuantizer`?

Reply:
Noted Thanks!